integrations/amazon_bedrock/examples/s3_downloader.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
Thanks for adding an example! I would like to expand it though to show when you would use this component in a pipeline. The major use case we are trying to enable is a Visual RAG QA pipeline in a production environment. 

So it would be great if we could use this tutorial as an example https://haystack.deepset.ai/tutorials/46_multimodal_rag and expand the query pipeline under the header **Multimodal RAG on Image + Text** to use this Downloader component.  The S3Downloader should probably come between `doc_type_router` and `doc_to_image` to ensure that the image file is available for `doc_to_image` to load it.